packages/labs/react/src/create-component.ts

```diff
@@ -128,15 +142,18 @@ const setProperty = <E extends Element>(
   events?: EventNames
 ) => {
   const event = events?.[name];
+  // Dirty check event value.
   if (event !== undefined && value !== old) {
-    // Dirty check event value.
     addOrUpdateEventListener(node, event, value as (e?: Event) => void);
     return;
   }
 
+  // But don't dirty check properties; elements are assumed to do this.
+  node[name as keyof E] = value as E[keyof E];
```

Comment:
All tests of expected behavior passes. I think it's a bit more correct to set the property to these nullish values then remove the attribute, rather than removing the attribute only and leaving the previous property values as it currently is.

Reply:
Any way we can add a test for what we think is the more correct behavior? I think it'll matter for properties that don't reflect from an attribute, or have a name that differs from the attribute.

Like for props: `{ariaChecked: null}`? The new behavior should be correct where the old was wrong.

Reply:
`ariaChecked` is kind of a weird one. React would normally not accept it as a valid prop on native elements and do nothing with it. It expects `aria-checked` as a prop to set that attribute, and if the user does that we'd never enter this logic and let React handle it.

Our wrapped component does accept `ariaChecked` as a prop since we find it in `HTMLElement.prototype` and set it. It's not quite the idiomatic React use case but since we're setting it we should remove. I can add a test for that.

Reply:
So turns out `el.ariaChecked = true` doesn't reflect to `aria-checked` in Firefox so users shouldn't really be doing that.
Our types disallow `ariaChecked` to be passed in as a prop and only accepts `aria-checked`. I'm now thinking we shouldn't have to add a test for something users shouldn't do.